packages/astro/src/integrations/index.ts

```diff
@@ -221,17 +221,21 @@ export async function runHookConfigDone({
 								logger
 							);
 							for (const [featureName, supported] of Object.entries(validationResult)) {
-								if (!supported) {
+								// If `supported` / `validationResult[featureName]` only allows boolean,
+								// in theory 'assets' false, doesn't mean that the feature is not supported, but rather that the chosen image service is unsupported
+								// in this case we should not show an error, that the featrue is not supported
+								// if we would refactor the validation to support more than boolean, we could still be able to differentiate between the two cases
+								if (!supported && featureName !== 'assets') {
 									logger.error(
 										'astro',
 										`The adapter ${adapter.name} doesn't support the feature ${featureName}. Your project won't be built. You should not use it.`
 									);
 								}
 							}
 							if (!validationResult.assets) {
-								logger.info(
+								logger.warn(
 									'astro',
-									`The selected adapter ${adapter.name} does not support Sharp or Squoosh for image processing. To ensure your project is still able to build, image processing has been disabled.`
+									`The selected adapter ${adapter.name} does not support image processing. To ensure your project continues to build successfully, the image service has been automatically switched to the 'noop' option, effectively disabling image processing.`
```

Comment:
```suggestion
									`The selected adapter ${adapter.name} does not support image processing. To allow your project to build with the original, unprocessed images, the image service has been automatically switched to the 'noop' option.`
```
I'm wondering whether something like this might be sufficient (and more reassuring that you do get images)? 

If it's possible to link for more info somewhere, then this would be a good place https://docs.astro.build/en/reference/configuration-reference/#imageservice (and, I did make myself a docs issue to document a noop option!)

Reply:
I don't think we should add a link to the terminal log, but your version sounds better.

Reply:
I've updated it & also added the URL, should be working.